lms/static/js/groups/collections/cohort.js

```diff
@@ -0,0 +1,16 @@
+var edx = edx || {};
+
+(function(Backbone, CohortModel) {
+    'use strict';
```

Comment:
@cahrens To your thoughts:
* I think the text size should match the radio button label. I'll adjust.
* I've made adjustments to the styling of the visual feedback messages in a previous commit.
* I'll play with tab size to make them a tiny bit smaller, but I rather like larger click targets.